src/components/list_group/list_group_item.tsx

```diff
@@ -144,6 +144,11 @@ export type EuiListGroupItemProps = CommonProps &
      * By default the text will be same as the label text.
      */
     toolTipText?: string;
+
+    /**
+     * Props can be passed to nested`EuiToolTip`.
+     */
+    toolTipProps?: EuiToolTipProps;
```

Comment:
This should be a partial
```suggestion
    toolTipProps?: Partial<EuiToolTipProps>;
```